cpu/esp32/startup.c

```diff
@@ -142,6 +142,10 @@ NORETURN void IRAM call_start_cpu0 (void)
         memset(&_rtc_bss_rtc_start, 0, (&_rtc_bss_rtc_end - &_rtc_bss_rtc_start));
     }
 
+#ifdef MODULE_DBGPIN
+    dbgpin_init();
+#endif
+
```

Comment:
This location does not seem to work well, I'm unsure where it should be. Maybe @gschorcht can suggest the best location?